src/components/common/side-menu/AnimationControls.js

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from '@material-ui/core/styles';
+import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+import PauseCircleOutlineIcon from '@material-ui/icons/PauseCircleOutline';
+import RotateLeftIcon from '@material-ui/icons/RotateLeft';
+import { green, yellow, orange } from '@material-ui/core/colors';
+import {
+  incrementIntervalCount,
+  reset,
+  setFinalCarbonDioxide,
+  setIsPaused,
+  setFinalMethane,
+  setFinalCloudCover,
+  setFinalIceCover,
+  resetFluxesFills,
+} from '../../../actions';
+import {
+  APPLICATION_INTERVAL,
+  GRADUAL_UPDATE_INTERVAL,
+  GRADUAL_UPDATE_NUM_INCREMENTS,
+} from '../../../config/constants';
+import {
+  graduallyDispatchCTerms,
+  graduallyDispatchIceCoverTerms,
+  graduallyDispatchValues,
+  stopFluxesBlinking,
+} from '../../../utils/canvas';
+import {
+  computeIceCoverFeedbackTerms,
+  computeWaterVaporFeedbackCTerms,
+} from '../../../utils/greenhouseEffect';
+import CustomButton from './shared-components/CustomButton';
+
+const useStyles = makeStyles(() => ({
+  buttonContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  button: { fontSize: '1.75em' },
+}));
+
+const AnimationControls = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const {
+    isPaused,
+    temporaryCarbonDioxide,
+    finalCarbonDioxide,
+    temporaryMethane,
+    finalMethane,
+    temporaryCloudCover,
+    temporaryIceCover,
+    finalIceCover,
+    finalCloudCover,
+    feedback,
+    cTerm,
+    simulationMode,
+  } = useSelector(({ lab }) => lab);
+  const { waterVaporFeedbackOn, iceCoverFeedbackOn } = feedback;
+  const applicationInterval = useRef();
+
+  const startInterval = () => {
+    applicationInterval.current = setInterval(() => {
+      dispatch(incrementIntervalCount());
+    }, APPLICATION_INTERVAL);
+  };
+
+  useEffect(() => {
+    if (isPaused) {
+      clearInterval(applicationInterval.current);
+    } else if (!isPaused) {
+      startInterval();
+    }
+  }, [isPaused]);
+
+  const onClickPlay = () => {
+    stopFluxesBlinking();
+    dispatch(resetFluxesFills());
+    dispatch(setIsPaused(false));
+
+    if (waterVaporFeedbackOn) {
+      dispatch(setFinalIceCover(temporaryIceCover));
+      dispatch(setFinalCloudCover(temporaryCloudCover));
+      dispatch(setFinalCarbonDioxide(temporaryCarbonDioxide));
+      dispatch(setFinalMethane(temporaryMethane));
+
+      const cTerms = computeWaterVaporFeedbackCTerms(
+        temporaryIceCover,
+        temporaryCloudCover,
+        temporaryCarbonDioxide,
+        temporaryMethane,
+        cTerm,
+        simulationMode,
+      );
+
+      graduallyDispatchCTerms(cTerms, dispatch, GRADUAL_UPDATE_INTERVAL);
+
+      return;
+    }
+
+    if (iceCoverFeedbackOn) {
```

Comment:
You never use both at the same time? It's not really clear to have `return` in the middle, maybe use `if-else if` or 
```js
switch(true) {
 case waterVaporFeedbackOn:
        // your code
       break;
 case iceCoverFeedbackOn:
        // your code
       break;
  default:
        // your code
       break;
}
```

Reply:
Good catch on using both at the same time. This case should be handled. #93